.github/workflows/release.yml

```diff
@@ -15,10 +15,11 @@ jobs:
         with:
           fetch-depth: 50
       - run: git tag ${{ github.event.inputs.Version }}
-      - uses: notlmn/release-with-changelog@v2
+      - uses: ./
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          exclude: '^Meta'
+          exclude: ^Meta
```

Comment:
IMO, lets have this as `'^Meta'` just to not confuse other users, and have this bite us back if something fails.

Reply:
```suggestion
          exclude: '^Meta'
```